src/storage/DisputeStorage.ts

```diff
@@ -62,6 +69,10 @@ export class DisputeStorage {
         return this.disputes.get(disputeHash);
     }
 
+    getDisputedFork(forkId: ForkId): DidIDispute {
```

Comment:
`didIDispute(forkId)` - so it's clear that only that is taken into account and not the existance of a `disputeWindow` + if you name it `getDisputedFork` it implies that the return value is a fork